biogeochem/EDPhysiologyMod.F90

```diff
@@ -1189,220 +1576,345 @@ subroutine CWD_Input( currentSite, currentPatch)
+                 trunk_wood * currentPatch%area * logging_export_frac
+
+            ! Add AG wood to litter from the non-exported fraction of wood 
+            ! from direct anthro sources
+
+            litt%ag_cwd_in(c) = litt%ag_cwd_in(c) +  &
+                 trunk_wood * (1._r8-logging_export_frac)
+
+            flux_diags%cwd_ag_input(c)  = flux_diags%cwd_ag_input(c) + &
+                 trunk_wood * (1._r8-logging_export_frac) * currentPatch%area
+
+            ! Add AG wood to litter from indirect anthro sources
+
+            litt%ag_cwd_in(c) = litt%ag_cwd_in(c) + (struct_m + sapw_m) * & 
+                 SF_val_CWD_frac(c) * (dead_n_natural+dead_n_ilogging)  * &
+                 EDPftvarcon_inst%allom_agb_frac(pft)
+
+            flux_diags%cwd_ag_input(c)  = flux_diags%cwd_ag_input(c) + &
+                  SF_val_CWD_frac(c) * (dead_n_natural+dead_n_ilogging) * &
+                  currentPatch%area * EDPftvarcon_inst%allom_agb_frac(pft)
+
          else
-            
-            currentPatch%cwd_AG_in(c) = currentPatch%cwd_AG_in(c) + (struct_c + sapw_c) * & 
+
+            litt%ag_cwd_in(c) = litt%ag_cwd_in(c) + (struct_m + sapw_m) * & 
                  SF_val_CWD_frac(c) * dead_n  * &
-                 EDPftvarcon_inst%allom_agb_frac(currentCohort%pft)
+                 EDPftvarcon_inst%allom_agb_frac(pft)
 
+            flux_diags%cwd_ag_input(c)  = flux_diags%cwd_ag_input(c) + &
+                  SF_val_CWD_frac(c) * dead_n * (struct_m + sapw_m) * &
+                  currentPatch%area * EDPftvarcon_inst%allom_agb_frac(pft)
+            
          end if
+         
+      end do
 
-         ! Update diagnostics that track resource management
+
+      ! Update diagnostics that track resource management
+
+      if( element_id .eq. carbon12_element ) then
+      
          currentSite%resources_management%delta_litter_stock  = &
               currentSite%resources_management%delta_litter_stock + &
-              (struct_c + sapw_c) * &
-              SF_val_CWD_frac(c) * (dead_n_natural+dead_n_ilogging) * & 
-              hlm_freq_day * currentPatch%area
-         
-         ! Update diagnostics that track resource management
+              (leaf_m + fnrt_m + store_m ) * &
+              (dead_n_ilogging+dead_n_dlogging) * currentPatch%area
+
          currentSite%resources_management%delta_biomass_stock = &
               currentSite%resources_management%delta_biomass_stock + &
-              (struct_c + sapw_c) * &
-              SF_val_CWD_frac(c) * dead_n * & 
-              hlm_freq_day * currentPatch%area
-         
-         if (currentPatch%cwd_AG_in(c) < 0.0_r8)then
-            write(fates_log(),*) 'negative CWD in flux',currentPatch%cwd_AG_in(c), &
-                 (struct_c + sapw_c), dead_n, dead_n_natural, dead_n_ilogging, dead_n_dlogging
-            call endrun(msg=errMsg(sourcefile, __LINE__))
-         endif
+              (leaf_m + fnrt_m + store_m ) * & 
```

Comment:
This part is hard to tell what is happening unless you open up the code for both old and new, and have them open next to each other.  I saw this particular diff, and I thought the pools going to the resources_management trackers were being changed, but after looking again, its not.  See this line up above:

https://github.com/NGEET/fates/pull/524/files#diff-b2580903c042bbb143bb267a7b882e3fL1309

Reply:
also, note that the units on the number of dead plants has changed, that is why I don't add hlm_freq_day to the calculations anymore, because the death rates are already in indiv/day.

https://github.com/NGEET/fates/pull/524/files#diff-b2580903c042bbb143bb267a7b882e3fR1723